src/main/java/com/bullhornsdk/data/api/helper/RestJsonConverter.java

```diff
@@ -84,19 +85,19 @@ private ObjectMapper createObjectMapperWithRootUnWrap() {
     }
 
     public <T> T jsonToEntity(String jsonString, Class<T> type, ObjectMapper objectMapper) {
-        T entity = null;
 
         try {
-            entity = objectMapper.readValue(jsonString, type);
+            T entity = objectMapper.readValue(jsonString, type);
+            return entity;
         } catch (JsonParseException e) {
             log.error("Error parsing jsonString to " + type + ". jsonString = " + jsonString, e);
```

Comment:
let's change this to look like

try {
   return objectMapper.readValue(jsonString, type);
} catch(JsonParseException | JsonMappingException | IOException e) {
  throw new RestMappingException("Error maping jsonString to " + type + ". jsonString = " + jsonString, e);
}

that way we pass the exception to the RestMappingException constructor and get information about what kind of exception occurred.